src/serviceFacades/groups.js

```diff
@@ -67,28 +62,28 @@ function searchTeams(key, { searchTerm }) {
     });
 }
 
-function listSingleTeam(key, { name }) {
+function listSingleTeam({ name }) {
     return callApi({
         endpoint: `/api/teams/${encodeURIComponent(name)}`,
         method: "GET",
     });
 }
 
-function getTeamPrivileges(key, { name }) {
+function getTeamPrivileges({ name }) {
     return callApi({
         endpoint: `/api/teams/${encodeURIComponent(name)}/privileges`,
         method: "GET",
     });
 }
 
-function getTeamMembers(key, { name }) {
+function getTeamMembers({ name }) {
     return callApi({
         endpoint: `/api/teams/${encodeURIComponent(name)}/members`,
         method: "GET",
     });
 }
 
-function getTeamDetails(key, { name }) {
+function getTeamDetails({ name }) {
     return Promise.all([
```

Comment:
The functions called by this Promise were all changed to only require the `{name}` param, so the key params need be removed below this.

Reply:
I could have sworn I made this change earlier🤔 😩 .